src/test/java/org/sagebionetworks/bridge/S3InitializerTest.java

```diff
@@ -101,8 +105,13 @@ public void bucketCreatedForPublicBucket() {
         
         verify(mockS3Client).createBucket(requestCaptor.capture());
         verify(mockS3Client).setBucketPolicy(BUCKET_NAME, resolvedPolicy);
+        verify(mockS3Client).setBucketWebsiteConfiguration(eq(BUCKET_NAME), websiteConfigCaptor.capture());
         
         assertEquals(requestCaptor.getValue().getBucketName(), BUCKET_NAME);
+        
+        // Not a lot to check here, but
+        BucketWebsiteConfiguration config = websiteConfigCaptor.getValue();
+        assertEquals(config.getIndexDocumentSuffix(), "index.html");
```

Comment:
Should we also check Cors configuration here?

Reply:
Added